code/modules/projectiles/projectile/magic/spellcard.dm

```diff
@@ -4,3 +4,60 @@
 	icon_state = "spellcard"
 	damage_type = BURN
 	damage = 2
+
+/obj/item/projectile/magic/spellcard/book
+	nodamage = FALSE
+	name = "enchanted page"
+	desc = "A piece of paper enchanted to give it extreme durability and stiffness, along with a very hot burn to anyone unfortunate enough to get hit by a charged one."
+	icon_state = "spellcard"
+	damage_type = BURN
+	damage = 12
+	flag = "magic"
+
+/obj/item/projectile/magic/spellcard/book/spark
+	damage = 4
+	var/fire_stacks = 4
```

Comment:
no need for this variable, just use the number.

Reply:
This var is not defined  tho